Cannot get ID for packet class net.md_5.bungee.protocol.packet.PlayerListItemRemove

### Bungeecord version

BungeeCord-1877 (Jenkins)

### Server version

1.21.1

### Client version

1.21.1

### Bungeecord plugins

```
ajQueuePlus-2.7.0
DomainStats-Bungee
floodgate-bungee-112
LuckPerms-Bungee-5.4.141
Nameless-Plugin-BungeeCord-3.4.0
nuvotifier
Pl-Hide-Pro-2.6.3
protocolize-bungeecord-2.3.1
RedirectPlus-2.12.0
ServerListMotd
spark-1.10.110-bungeecord
tebex-bungeecord-2.1.0
UltraStaffChat-5.2.2
```

### The bug

Players get kicked, seemingly at random, with the attached error message. I've removed every bungeecord plugin and run the proxy on nothing, with the kick errors when players connect, or switch worlds, still occurring. I'm at a complete loss now, and no other devs, system admins, or server owners seem to know the cause either.
![image](https://github.com/user-attachments/assets/a0f7983d-f47a-40c8-90fd-2af260d3f1da)


### Log output (links)

```16:41:07 [SEVERE] [InLaLaLand13] -> UpstreamBridge - encountered exception
io.netty.handler.codec.EncoderException: java.lang.IllegalArgumentException: Cannot get ID for packet class net.md_5.bungee.protocol.packet.PlayerListItemRemove in phase CONFIGURATION with direction TO_CLIENT
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:125)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:893)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite(AbstractChannelHandlerContext.java:875)
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:984)
	at io.netty.channel.AbstractChannelHandlerContext.write(AbstractChannelHandlerContext.java:868)
	at io.netty.handler.codec.MessageToMessageEncoder.write(MessageToMessageEncoder.java:113)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:893)
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:956)
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1263)
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:173)
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:166)
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:469)
	at io.netty.channel.epoll.EpollEventLoop.run(EpollEventLoop.java:405)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:994)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at java.base/java.lang.Thread.run(Thread.java:840)
Caused by: java.lang.IllegalArgumentException: Cannot get ID for packet class net.md_5.bungee.protocol.packet.PlayerListItemRemove in phase CONFIGURATION with direction TO_CLIENT
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:463)
	at net.md_5.bungee.protocol.Protocol$DirectionData.getId(Protocol.java:992)
	at net.md_5.bungee.protocol.MinecraftEncoder.encode(MinecraftEncoder.java:26)
	at net.md_5.bungee.protocol.MinecraftEncoder.encode(MinecraftEncoder.java:10)
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107)
	... 15 more```

### Checking

- [X] I am using BungeeCord and **not a fork**. Issues with forks should not be reported here.
- [X] I think this is **not** an issue with a bungeecord plugin.
- [ ] I have not read these checkboxes and therefore I just ticked them all.
- [X] This is not a question or plugin creation help request.

--- a/proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java
+++ b/proxy/src/main/java/net/md_5/bungee/connection/UpstreamBridge.java
@@ -101,7 +101,8 @@ public class UpstreamBridge extends PacketHandler
             {
                 if ( player.getPendingConnection().getVersion() >= ProtocolConstants.MINECRAFT_1_19_3 )
                 {
-                    player.unsafe().sendPacket( newPacket );
+                    // need to queue, because players in config state could receive it
+                    ( (UserConnection) player ).sendPacketQueued( newPacket );
                 } else
                 {
                     player.unsafe().sendPacket( oldPacket );

